src/components/AddContent/AddContent.test.tsx

```diff
@@ -119,6 +126,91 @@ it('expect "url" input to show a validation error', async () => {
   expect(queryByText('Invalid URL')).toBeInTheDocument();
 });
 
+it('expect "Package and metadata verification" to be pre-selected', async () => {
+  (useAddContentQuery as jest.Mock).mockImplementation(() => ({ isLoading: false }));
+  (useValidateContentList as jest.Mock).mockImplementation(() => ({
+    isLoading: false,
+    mutateAsync: async () => passingValidationErrorData,
+    data: passingValidationErrorData,
+  }));
+  (useDebounce as jest.Mock).mockImplementation((value) => value);
+
+  const { queryByText, queryByPlaceholderText, queryByLabelText } = render(
+    <ReactQueryTestWrapper>
+      <AddContent />
+    </ReactQueryTestWrapper>,
+  );
+
+  const button = queryByText('Add repositories');
+  expect(button).toBeInTheDocument();
+  await act(async () => {
+    button?.click();
+  });
+  const urlInput = queryByPlaceholderText('https://');
+  expect(urlInput).toBeInTheDocument();
+  if (urlInput) {
+    await act(async () => {
+      fireEvent.change(urlInput, { target: { value: 'https://bobTheBuilder.com' } });
+      fireEvent.blur(urlInput);
+    });
+  }
+  expect(queryByText('Invalid URL')).not.toBeInTheDocument();
+  const gpgKeyInput = queryByPlaceholderText('Paste GPG key or URL here');
+  expect(gpgKeyInput).toBeInTheDocument();
+  if (gpgKeyInput) {
+    await act(async () => {
+      fireEvent.change(gpgKeyInput, { target: { value: 'aRealGPGKey' } });
+    });
+  }
+
+  expect(queryByText('Package and metadata verification')).toBeInTheDocument();
+  const packageMetaDataInput = queryByLabelText('Package and metadata verification');
+
+  expect(packageMetaDataInput).toHaveAttribute('checked');
+});
+
+it('expect "Package verification only" to be pre-selected', async () => {
+  (useAddContentQuery as jest.Mock).mockImplementation(() => ({ isLoading: false }));
+
+  (useValidateContentList as jest.Mock).mockImplementation(() => ({
+    isLoading: false,
+    mutateAsync: async () => passingValidationMetaDataSigNotPresent,
+    data: passingValidationMetaDataSigNotPresent,
+  }));
+  (useDebounce as jest.Mock).mockImplementation((value) => value);
+
+  const { queryByText, queryByPlaceholderText, queryByLabelText } = render(
+    <ReactQueryTestWrapper>
+      <AddContent />
+    </ReactQueryTestWrapper>,
+  );
+
+  const button = queryByText('Add repositories');
+  expect(button).toBeInTheDocument();
+  await act(async () => {
+    button?.click();
+  });
+  const urlInput = queryByPlaceholderText('https://');
+  expect(urlInput).toBeInTheDocument();
+  if (urlInput) {
+    await act(async () => {
+      fireEvent.change(urlInput, { target: { value: 'https://bobTheBuilder.com' } });
+    });
+  }
+  expect(queryByText('Invalid URL')).not.toBeInTheDocument();
```

Comment:
i guess there's no way to reference the text via some constant rather than duplicating it?  I only say this because if the text changes, this test would still keep on passing even though its really not testing anything (and wouldn't detect a failure)

Reply:
Oh like a global constants for validation errors? We could do that.

Reply:
if it makes sense to do that more globally, lets file a ticket (i'm happy to do that)